src/app/storymaps/tpl/builder/settings/ViewMapOptions.html

```diff
@@ -36,4 +36,23 @@
 			</a>
 		</label>
 	</div>
+      
+    <div class="" style="">
+      <label>
+        Maptiks Trackcode: <input type="text" class="form-control input-sm maptiks-trackcode" />
```

Comment:
lets change this to use the 'id' attribute rather than the 'class'. 

ie: <input id="maptiks-trackcode">